__integration__/__snapshots__/customFormats.test.snap.js

```diff
@@ -962,7 +962,8 @@ snapshots["inline custom with new args should match snapshot"] =
   },
   "options": {
     "otherOption": "Test",
-    "showFileHeader": true
+    "showFileHeader": true,
+    "usesW3C": false
```

Comment:
I wonder if this should be `usesDtcg` instead. The spec at is not a W3C one. A working draft is not a W3C standard (and there is a small chance that it’ll never be one). If I understand correctly, usage of ‘W3C‘ in code or comments is misleading and discouraged. Related discussion: https://github.com/design-tokens/community-group/issues/164

Reply:
Thanks, that makes sense! I've adjusted the PR and also got rid of any other W3C term occurrences.

In the debate of "userId" versus "userID", I'm on the side of "userId" since fairly recently, so using all-caps for such abbreviations is still a bit engrained in my habits sometimes 😅

Reply:
Yeah `strictCamelCase` vs `looseCAMELCase` is a simpler rule to follow. Even though it may produce vars that are odd-looking at first glance (e.g. `myUiConfig`), using the rule everywhere frees up some cognitive resources. There is no need to think through each individual case.